### PR TITLE
Created Job_Sequencing in java 

### PR DESCRIPTION
**Pull request by papri24majumder**
## 🛠️ Issue (Number)

Issue no #282

## ✔️ Check List (Check all the applicable boxes) 

- [✔️ ] My code follows the code style of this project.
- [✔️ ] This PR does not contain plagiarized content.
- [ ✔️] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

- It is considered that each job takes 1 unit of time.
- The jobs are arranged in descending order.

## 📷 Screenshots

image

### CODE REVIEW

**Review by zugzwang03 — APPROVED** [review]

File: Java/Greedy-Algorithm/Job_Sequencing.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    }
+
+    // Arranging jobs in descending order if not sorted
+    void sort() {
+        int temp;
+        String temp1;
+        for (int i = 0; i < n - 1; i++) {
+            for (int j = i + 1; j < n; j++) {
+                if (b[i] < b[j]) {
+                    temp = b[i];
```
Comment (Rancho2002) [inline Java/Greedy-Algorithm/Job_Sequencing.java:56]: create a swap function instead of writing same code again and again

**Review by Rancho2002 — CHANGES_REQUESTED** [review]

**Review by 072arushi — CHANGES_REQUESTED** [review]
Add comments to explain your algorithm

File: Java/Greedy-Algorithm/Job_Sequencing.java (hunk 1 above)
Comment (papri24majumder) [reply Java/Greedy-Algorithm/Job_Sequencing.java:56]: @Rancho2002 creating a swap function is giving me errors. So, can I keep it the same?

Comment (Rancho2002) [reply Java/Greedy-Algorithm/Job_Sequencing.java:57]: okay then its all right.

**Review by Rancho2002 — APPROVED** [review]

**Comment by papri24majumder** [comment]
@072arushi Done. Please check

**Review by 072arushi — APPROVED** [review]

**Review by Sanjanabharadwaj25 — APPROVED** [review]

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**